docs/index.rst

```diff
@@ -111,54 +113,75 @@ function.
 
 .. code-block:: python
 
+    import flask
+    from flask_login import login_user
+
     @app.route('/login', methods=['GET', 'POST'])
     def login():
         # Here we use a class of some kind to represent and validate our
         # client-side form data. For example, WTForms is a library that will
         # handle this for us, and we use a custom LoginForm to validate.
         form = LoginForm()
+
         if form.validate_on_submit():
             # Login and validate the user.
             # user should be an instance of your `User` class
             login_user(user)
 
             flask.flash('Logged in successfully.')
 
-            next = flask.request.args.get('next')
+            next_url = form.next_url.data
```

Comment:
`next_url` is good naming but keep it coming from `flask.request.args.get('next')` since this doesn't belong in the login form and this breaks the default unauthorized handler which names the request arg `next` so the form would always see the next url as missing.

Reply:
For this example to work, does the action of your login form have to be blank? Because if it is not then using `flask.request.args.get('next')` here doesn't work. If it supposed to be blank, that is very important information that is missing that I can add.